api/swagger.yaml

```diff
@@ -8768,16 +8768,6 @@ paths:
                   type: "string"
                 is_official:
                   type: "boolean"
-                is_automated:
-                  description: |
-                    Whether this repository has automated builds enabled.
-
-                    <p><br /></p>
-
-                    > **Deprecated**: This field is deprecated and will always
-                    > be "false" in future.
-                  type: "boolean"
-                  example: false
```

Comment:
We can keep the field (removing it was considered more problematic for clients that don't "unmarshal" but directly use the response), but it would always be "false"

Reply:
Ah yeah, deleted too much here. Restored this one.